### PR TITLE
Added CSS file 

### PR DESCRIPTION
**Pull request by hitg23**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+body {
+background-color: aquamarine;
```
Comment (yayner2002) [inline style.css:2]: Good job so far. :+1:  :100:

File: style.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+font-size: medium;
+}
```
Comment (yayner2002) [inline style.css:4]: nice work. but I kindly ask you to add more CSS to the body tag. this will help you to ......

**Review by yayner2002 — APPROVED** [review]
https://media.tenor.com/BPubfqCehIsAAAAC/great-job-good-job.gif
Good Job @hitg23  and @ZerituMe. Your project is completed. so now it's time to merge your branch. :1st_place_medal:

File: style.css (hunk 1 above)
Comment (hitg23) [reply style.css:2]: thank you so much I accept the comment 🥇

**Pull request merged**